index.rst

```diff
@@ -21,6 +21,8 @@ Contents:
    Scriptworker <https://scriptworker.readthedocs.io/en/latest/index.html>
    Scriptworker Scripts <https://scriptworker-scripts.readthedocs.io/en/latest/index.html>
    Treeherder <https://treeherder.readthedocs.io/>
+   Accessing CloudOps' Jenkins <https://github.com/mozilla-services/cloudops-deployment#accessing-jenkins>
```

Comment:
This 404s for me.

Reply:
It's a private repo, which I guess you don't have access to.